### ISSUE TITLE
Error running tests - mocha not found

### ISSUE THREAD

**Issue by kevinburke** [pubref/rules_node#23]
I'm getting the following error when running `bazel test --test_output=errors //...`:

```
BUILD.bazel:8:1: every rule of type mocha_test implicitly depends upon the target '@npm_mocha//:bin/mocha', but this target could not be found because of: no such package '@npm_mocha//': The repository could not be resolved.
```

Here's my WORKSPACE:

```
git_repository(
    name = "org_pubref_rules_node",
    remote = "https://github.com/pubref/rules_node.git",
    commit = "6a213a61055c8c92097b8b8c1a7eaa5a86557b8a",
)

load("@org_pubref_rules_node//node:rules.bzl", "node_repositories")

node_repositories()
```

Here's my BUILD.bazel:

```
load("@org_pubref_rules_node//node:rules.bzl", "node_library", "mocha_test")

node_library(
    name = "gostdjs",
    srcs = glob(["**/*.js", "**/*.test.js", "0"]),
)

mocha_test(
    name="stdlib_tests",
    main="strings/strings.test.js",
    modules = [],
    deps = [],
    mocha_args = [
        "--slow=2",
    ]
)
```

Not sure what I'm doing wrong...

**Comment by kevinburke** [comment]
Ah, I need this dependency I think:

```
npm_repository(
    name = "npm_mocha",
    deps = {
        "mocha": "3.5.0",
    },
    #sha256 = "9b48987065bb42003bab81b4538afa9ac194d217d8e2e770a5cba782249f7dc8",
)
```

(The EXAMPLE that has a mocha test should also have this)

**Comment by pcj** [comment]
Thanks @kevinburke, closed by #26.
